packages/@okta/vuepress-site/docs/guides/terraform-enable-org-access/main/index.md

```diff
@@ -91,6 +91,19 @@ If the key isn't in the right format, convert it to the correct format using the
 
 > **Note:** The file that contains the converted private key must begin with `-----BEGIN RSA PRIVATE KEY-----`. If not, try step two again.
 
+## Assign admin roles
```

Comment:
@mauricesharp-okta , @MikeMondragon-okta: this feature doesn't apply to all orgs. You can confirm with @brentschaus-okta and the ACE team.

What orgs are terraform users using for this example? Maybe you can add a note in the beginning, like:

> **Note:** This step is only required for Preview orgs. For Production orgs, this step is required only if the **Assign admin roles to public client apps** early access feature is enabled.